index.html

```diff
@@ -1075,6 +1078,115 @@ <h2>Status List Verifiable Credential</h2>
 }
 </pre>
       </section>
+
+      <section>
+        <h2>Multiple Status Lists in One Verifiable Credential</h2>
+
+        <p>
+This specification enables an <a>issuer</a> to associate multiple status lists
+with a single <a>verifiable credential</a>.
+        </p>
+
+        <pre class="example nohighlight vc"
+             title="Associating multiple status lists with a single Verifiable Credential"
+             data-vc-vm='https://example.edu/issuers/565049/keys/1'>
```

Comment:
Is there a reason that `'` is used to wrap `data-vc-vm` values (lines 1092 and 1157), instead of `"` as on all other attribute values? It just seems odd to differ like this...

Reply:
No, no reason, just a typo.

Reply:
Fixed in 8d234dfed27adc8c84ad49ce1b7da28c41ca39ab